tdrs-backend/.env.example

```diff
@@ -0,0 +1,38 @@
+### Local Development Environment
+# Copy this file to `.env` and replace variables as needed
+#
+DISABLE_COLLECTSTATIC=1
+
+# Parameters defined in the login.gov documentation https://developers.login.gov/oidc/
+CLIENT_ID=urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-proto-dev
+OIDC_RP_CLIENT_ID=urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-proto-dev
+ACR_VALUES=http://idmanagement.gov/ns/assurance/ial/1
+OIDC_OP_ISSUER=https://idp.int.identitysandbox.gov/
+OIDC_OP_JWKS_ENDPOINT=https://idp.int.identitysandbox.gov/api/openid_connect/certs
+OIDC_OP_AUTHORIZATION_ENDPOINT=https://idp.int.identitysandbox.gov/openid_connect/authorize
+OIDC_OP_LOGOUT_ENDPOINT=https://idp.int.identitysandbox.gov/openid_connect/logout
+OIDC_OP_USER_ENDPOINT=https://idp.int.identitysandbox.gov/api/openid_connect/userinfo
+CLIENT_ASSERTION_TYPE=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+
+# Local django settings to define the execution environment
+DJANGO_SETTINGS_MODULE=tdpservice.settings.local
+DJANGO_CONFIGURATION=Local
+
+# Sets the URL values used for redirects
+FRONTEND_BASE_URL=http://localhost:3000
+BASE_URL=http://localhost:8080/v1
+
+# mock token for unit testing
+# saving here to resolve linting error for string too long
+MOCK_TOKEN=eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJiMmQyZDExNS0xZDdlLTQ1NzktYjlkNi1mOGU4NGY0ZjU2Y2EiLCJpc3MiOiJodHRwczovL2lkcC5pbnQubG9naW4uZ292IiwiYWNyIjoiaHR0cDovL2lkbWFuYWdlbWVudC5nb3YvbnMvYXNzdXJhbmNlL2xvYS8xIiwibm9uY2UiOiJhYWQwYWE5NjljMTU2YjJkZmE2ODVmODg1ZmFjNzA4MyIsImF1ZCI6InVybjpnb3Y6Z3NhOm9wZW5pZGNvbm5lY3Q6ZGV2ZWxvcG1lbnQiLCJqdGkiOiJqQzdOblU4ZE5OVjVsaXNRQm0xanRBIiwiYXRfaGFzaCI6InRsTmJpcXIxTHIyWWNOUkdqendsSWciLCJjX2hhc2giOiJoWGpxN2tPcnRRS196YV82dE9OeGN3IiwiZXhwIjoxNDg5Njk0MTk2LCJpYXQiOjE0ODk2OTQxOTgsIm5iZiI6MTQ4OTY5NDE5OH0.pVbPF-2LJSG1fE9thn27PwmDlNdlc3mEm7fFxb8ZADdRvYmDMnDPuZ3TGHl0ttK78H8NH7rBpH85LZzRNtCcWjS7QcycXHMn00Cuq_Bpbn7NRdf3ktxkBrpqyzIArLezVJJVXn2EeykXMvzlO-fJ7CaDUaJMqkDhKOK6caRYePBLbZJFl0Ri25bqXugguAYTyX9HACaxMNFtQOwmUCVVr6WYL1AMV5WmaswZtdE8POxYdhzwj777rkgSg555GoBDZy3MetapbT0csSWqVJ13skWTXBRrOiQQ70wzHAu_3ktBDXNoLx4kG1fr1BiMEbHjKsHs14X8LCBcIMdt49hIZg
+
+# Ask the project Tech Lead or Product Manager for the development credentials document for the rest of these values
```

Comment:
My understanding is that the only secrets a new developer will need in order to run the project are `JWT_KEY` and `JWT_CERT_TEST`. 

`DJANGO_SECRET_KEY` and `SECRET_KEY` don't need to be set to any particular value locally. [The docs recommend "a large random value" for production](https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/#secret-key) but locally, any random value should do.

`OIDC_OP_TOKEN_ENDPOINT` isn't a secret. 

So maybe this "Ask the Tech Lead" line can be moved down to just apply to the JWT secrets.

Reply:
Also, taking a look at both, it looks like we may not need a `SECRET_KEY` ENV variable at all, just a `DJANGO_SECRET_KEY`. 

The `DJANGO_SECRET_KEY` ENV variable gets read in as `SECRET_KEY` in `common.py`.

Reply:
Makes sense. I've updated accordingly.